spaceship/lib/spaceship/connect_api/models/profile.rb

```diff
@@ -92,6 +92,12 @@ def fetch_all_devices(client: nil, filter: {}, includes: nil, sort: nil)
         return resps.flat_map(&:to_models)
       end
 
+      def fetch_all_certificates(client: nil, filter: {}, includes: nil, sort: nil)
```

Comment:
Logically should this not be in certificates model? (https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/connect_api/models/certificate.rb)

Reply:
in `Certificate.all` method we fetch all certificates ([doc](https://developer.apple.com/documentation/appstoreconnectapi/list_and_download_certificates)).
Here we want only to fetch certificates for selected profile ([doc](https://developer.apple.com/documentation/appstoreconnectapi/list_all_certificates_in_a_profile))